core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXInternalV1Endpoint.java

```diff
@@ -36,6 +38,12 @@ public interface DistroXInternalV1Endpoint {
             nickname = "getDistroXInternalV1ByCrn")
     StackViewV4Response getByCrn(@PathParam("crn") String crn);
 
+    @GET
+    @Path("crn/{crn}/instances")
+    @ApiOperation(value = GET_BY_CRN_INSTANCES_INTERNAL, produces = MediaType.APPLICATION_JSON, notes = Notes.STACK_NOTES,
+            nickname = "getDistroXInternalV1ByCrn")
+    StackInstancesV4Responses getInstancesByCrn(@PathParam("crn") String crn);
```

Comment:
I wonder why we do not have `@ValidCrn(resource = {CrnResourceDescriptor.DATAHUB})` for `crn` anywhere in this class.